src/App.js

```diff
@@ -1,15 +1,33 @@
 import './App.css';
-// import images from './data.js';
+import animalData from './data.js';
```

Comment:
nice, I like that you statically rendered your page before implement javascript, so that you could break it down into smaller pieces